cray_product_catalog/catalog_update.py

```diff
@@ -214,7 +221,15 @@ def main():
         )
 
     load_k8s()
-    data = read_yaml_content(YAML_CONTENT)
+    if YAML_CONTENT:
+        data = read_yaml_content(YAML_CONTENT)
+    elif RAW_YAML_CONTENT:
+        data = read_yaml_content_string(RAW_YAML_CONTENT)
+    else:
+        LOGGER.error(
+            "Either YAML_CONTENT or RAW_YAML_CONTENT must be specified."
```

Comment:
Small nitpick on this error message, it might be better to specify that it's the environment variables `YAML_CONTENT` and `RAW_YAML_CONTENT` that need to be set.

Reply:
Done, thanks!